test/ably/restpush_test.py

```diff
@@ -222,6 +223,35 @@ def test_admin_device_registrations_remove_where(self):
         # Remove with no matching params
         assert self.remove_device_where(clientId=device.client_id).status_code == 204
 
+    # RSH1c1
+    def test_admin_channel_subscriptions_list(self):
+        list_ = self.ably.push.admin.channel_subscriptions.list
+
+        channel = 'canpublish:test1'
+
+        # Register several channel subscriptions for later use
+        save = self.ably.push.admin.channel_subscriptions.save
+        for key in self.devices:
+            device = self.devices[key]
+            save(PushChannelSubscription(channel, device_id=device.id))
+
+        response = list_(channel=channel)
+        assert type(response) is PaginatedResult
+        assert type(response.items) is list
+        assert type(response.items[0]) is PushChannelSubscription
+
+        # limit
+        assert len(list_(channel=channel, limit=5000).items) == len(self.devices)
+        assert len(list_(channel=channel, limit=2).items) == 2
+
+        # Filter by device id
+        device = self.get_device()
+        assert len(list_(channel=channel, deviceId=device.id).items) == 1
```

Comment:
It'd be nice to check also that the `device.id` matches the one saved in https://github.com/ably/ably-python/pull/120/files#diff-e2eb6519254bd4b1eb7b9c3357be80c6R236